### DISCUSSION TITLE
Can I use the refresh rate monitor?

### DISCUSSION THREAD

**Discussion by LitterMin** [tribalfs/GalaxyMaxHzPub#244] [Ideas]
![Screenshot_20241109_224842_Galaxy MaxHz.png](https://github.com/user-attachments/assets/60c466e3-e9aa-4297-99a2-4a6763fdc73d)

8.1.90Beta can no longer display with refresh rate 😭

**Comment by tribalfs** [comment]
Just click it again.

**Reply by LitterMin** [reply]
I've been unable to click on it,This content shows that it needs to use the system's own display refresh rate
Has this feature been canceled?🤔

**Reply by tribalfs** [reply]
Click OK on the prompt.

**Reply by LitterMin** [reply]
![Screenshot_20241114_223437_Galaxy MaxHz.png](https://github.com/user-attachments/assets/5bc8a0b9-3273-49e4-85f5-d188cd17d831)

Tip Because the Android version cannot use the real-time refresh rate🤔

**Reply by tribalfs** [reply]
You should update to [v8.1.90-beta4](https://github.com/tribalfs/GalaxyMaxHzPub/releases/download/v8.1/galaxy_max_hz_v8.1.90-beta.apk). Then click this.

![Screenshot_20241115_013045_Galaxy MaxHz](https://github.com/user-attachments/assets/7bb90b37-cf22-4605-ac69-2272560c9f25)
